README.rst

```diff
@@ -32,21 +32,49 @@ Javascript. With Batavia, you can run Python bytecode in your browser.
 It honors Python 3.4.4+ syntax and conventions, and let's you
 reference objects and classes defined natively in JavaScript.
 
-Tutorial
---------
+Quick Start
+---------------
 
-To take Batavia for a spin, run through the `Getting Started guide`_.
+(For a more detailed setup, please see the `Getting Started tutorial <https://batavia.readthedocs.io/en/latest/tutorials/tutorial-0.html>`_)
 
-Then have some fun with `the first tutorial`_, and try out running Python in
-your browser.
+Batavia requires a Python 3.4 installation, a copy of `Ouroboros <https://github.com/pybee/ouroboros>`_, and a virtualenv to run it all in. 
 
-.. _Getting Started guide: https://batavia.readthedocs.io/en/latest/intro/getting-started.html
-.. _the first tutorial: https://batavia.readthedocs.io/en/latest/tutorials/tutorial-0.html
+Clone the code repos:: 
+
+   $ mkdir pybee
+   $ cd pybee
+   $ git clone https://github.com/pybee/batavia
+   $ git clone https://github.com/pybee/ouroboros
+
+Setup a virtualenv (for other environments, see `Getting Started <https://batavia.readthedocs.io/en/latest/tutorials/tutorial-0.html>`_)::
```

Comment:
I followed the link for other environments and then got a bit lost because the link is to the root of the tutorial, if there's a way to painlessly semi-deep-link to a relative path of the section that describes the environments or something similar, it would be helpful.